src/features/events/components/EventParticipansTab/EventParticipantsList.tsx

```diff
@@ -0,0 +1,204 @@
+      renderCell: (params) => {
+        if (params.row.person) {
+          return <Typography>{''}</Typography>;
+        } else {
+          return <Typography>{params.row.phone}</Typography>;
+        }
+      },
+    },
+    {
+      disableColumnMenu: true,
+      field: 'email',
+      flex: 1,
+      headerName: messages.eventParticipantsList.columnEmail(),
+      hideSortIcons: true,
+      valueGetter: (params) => {
+        if (params.row.person) {
+          return '';
+        } else {
+          return `${params.row.email}`;
+        }
+      },
+    },
+    {
+      disableColumnMenu: true,
+      field: 'notified',
+      flex: 1,
+      headerName: messages.eventParticipantsList.columnNotified(),
+      hideSortIcons: true,
+      renderCell: (params) => {
+        if (params.row.person) {
+          return <ZUIRelativeTime datetime={params.row.response_date} />;
+        } else {
+          return <ZUIRelativeTime datetime={params.row.reminder_sent} />;
+        }
+      },
+    },
+    {
+      align: 'right',
+      disableColumnMenu: true,
+      field: 'cancel',
+      flex: 1,
+      headerName: '',
+      hideSortIcons: true,
+      renderCell: (params) => {
+        if (params.row.person) {
+          return (
+            <>
+              <Button sx={{ marginRight: '10px' }} variant="text">
+                {messages.eventParticipantsList.buttonCancel()}
+              </Button>
+              <Button
+                onClick={noPropagate(() => {
+                  model.addParticipant(params.row.id);
+                })}
+                variant="outlined"
+              >
+                {messages.eventParticipantsList.buttonBook()}
+              </Button>
+            </>
+          );
+        } else {
+          return (
+            <Button variant="text">
+              {messages.eventParticipantsList.buttonCancel()}
+            </Button>
+          );
+        }
+      },
+    },
+  ];
+
+  return (
+    <Box>
+      {model.getSignedParticipants() > 0 && (
+        <>
+          <Box
+            sx={{
+              '& div': { backgroundColor: 'transparent' },
+              alignItems: 'center',
+              display: 'flex',
+              flexDirection: 'row',
+              marginBottom: '15px',
+              marginTop: '15px',
+            }}
+          >
+            <Typography mr={2} variant="h4">
+              {messages.eventParticipantsList.signUps()}
+            </Typography>
+            <ZUINumberChip
+              color={theme.palette.grey[500]}
+              outlined={true}
+              value={model.getSignedParticipants()}
+            />
+          </Box>
+          <Typography mb={2} variant="body1">
+            {messages.eventParticipantsList.descriptionSignups()}
+          </Typography>
+          <DataGridPro
+            autoHeight
+            checkboxSelection
+            columns={columns}
+            rows={model.getPendingSignUps() ?? []}
+          />
```

Comment:
It looks to me like this could be made a reusable component that is used twice (eventually three times) on this page. Does that make sense?